clients/da-vinci-client/src/main/java/com/linkedin/davinci/store/AbstractStoragePartition.java

```diff
@@ -175,4 +176,8 @@ public void deleteWithReplicationMetadata(byte[] key, byte[] metadata) {
   public long getRmdByteUsage() {
     throw new VeniceUnsupportedOperationException("getRmdByteUsage");
   }
+
+  public RocksIterator getRocksDBIterator() {
```

Comment:
We cannot do this like this.  AbstractStorage* is meant to abstract the underlying storage.  It shouldn't leak Rocksdb interfaces through it.

Reply:
Should I create an `AbstractStorageIterator` then create a RocksDB implementation of it?